bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gems/Gems.java

```diff
@@ -166,6 +166,9 @@ public double getAOCorrectedFWHM(boolean doThrow) {
                                 return 0.12;
                         }
                         break;
+
+                    default:
+                        throw new IllegalArgumentException("GeMS does not support band " + strehlBand);
```

Comment:
For reasons unknown to me the current ITC implementation only supports J, H and K bands in some of the calculations. This error should actually not be thrown since on the web page you can only select one of those bands and with the current implementation in the OT it is also guaranteed to have only one of these bands being sent down into the ITC code. However I thought it would be a good idea to add an exception for the future so that it is explicit what's going on in case somebody sends other bands down because the fall back would have been to get an exception with the message "Gems can not be used with IQ any" which is arguably even worse. I think.